app/helpers/application_helper.rb

```diff
@@ -1,9 +1,9 @@
 module ApplicationHelper
   def display_picture(game)
-    if game.picture.nil?
-      image_tag('app/assets/images/fallback/default.png')
+    if game.pictures.length.zero?
+      image_tag('fallback/default.png')
     else
```

Comment:
Should probably keep the check for nil pictures
'game.pictures&.length.zero?' should do the trick